spec/models/metric/statistic_spec.rb

```diff
@@ -5,7 +5,7 @@
                          :zone => Zone.first)
     end
 
-    hour = Time.parse(Metric::Helper.nearest_hourly_timestamp(Time.now)).utc
+    hour = Time.now.utc.change(:min => 0)
```

Comment:
I got rid of it where I could.
There are a few instances that pass in a string.
It felt like it would take some understanding and effort to track all of those down